samples/writeBatch.js

```diff
@@ -70,7 +70,9 @@ function main(instanceId = 'YOUR_INSTANCE_ID') {
     await table.insert(rowsToInsert);
 
     console.log(
-      `Successfully wrote 2 rows: ${rowsToInsert[0].key} and ${rowsToInsert[1].key}`
+      `Successfully wrote 2 rows: ${rowsToInsert[0].key} and ${
```

Comment:
You may need to do a `rm -rf node_modules package-lock.json && npm install && npm run fix`

Reply:
did that, not sure if that fixed what you wanted